docker-compose.yml

```diff
@@ -1,17 +1,23 @@
 version: "3"
 services:
-  create-and-vary-a-licence-api:
-    build:
-      context: .
-    network_mode: "host"
-    container_name: create-and-vary-a-licence-api
+  licences-db:
+    image: postgres
```

Comment:
good to specify the [version](https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/create-and-vary-a-licence-api-prod/resources/rds.tf#L13) we use in prod here?